Makefile

```
@@ -1,4 +1,5 @@
 FILE=./VERSION
 RESUMIC_VERSION = `cat $(FILE)`
+GITCOMMIT = `git describe --tags --always --dirty`
 build: ./cmd/version.go
-	go build -ldflags "-X github.com/resumic/schema/cmd.Version=$(RESUMIC_VERSION)" -o resumic
+	go build -ldflags "-X github.com/resumic/schema/cmd.Version=$(RESUMIC_VERSION) -X github.com/resumic/schema/cmd.GitCommit=$(GITCOMMIT)" -o resumic
```

```suggestion
	go build -ldflags "-X $(go list -m)/cmd.Version=$(RESUMIC_VERSION) -X $(go list -m)/cmd.GitCommit=$(GITCOMMIT)" -o resumic
```

This should make the import path non hardcoded and retrieved dynamically.